src/lib.rs

```diff
@@ -260,6 +260,8 @@ pub use os::OsRng;
 
 pub use isaac::{IsaacRng, Isaac64Rng};
 pub use chacha::ChaChaRng;
+#[deprecated(since="0.3.18", note="renamed to seq::sample_iter")]
```

Comment:
That's the current version; I *guess* this should be `0.3.19` with a corresponding version bump in Cargo.toml.